src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/BasePostsCreator.java

```diff
@@ -43,8 +47,16 @@ public BasePostsCreator() {
 
   @Override
   public PostsCreator click() {
-    wait.forElementClickable(getPostsCreator());
-    getPostsCreator().click();
+    wait.forElementClickable(textPostType);
+    textPostType.click();
+
+    /*
+      This entry point is disabled until the outcome of experiment in IRIS-5829
+
+      wait.forElementClickable(getPostsCreator());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)